interface/interface.dm

```diff
@@ -65,7 +65,8 @@
 			message += GLOB.revdata.GetTestMergeInfo(FALSE)
 		if(tgalert(src, message, "Report Issue","Yes","No")=="No")
 			return
-		src << link("[githuburl]/issues/new")
+		var/static/issue_template = url_encode(file2text(".github/ISSUE_TEMPLATE.md"))
+		src << link("[githuburl]/issues/new[GLOB.round_id ? "?body=Issue reported from Round ID: [GLOB.round_id][url_encode("\n\n")][issue_template]" : ""]")

```

Comment:
actually, the spaces would also need url encoding. I think it still technically works with them, but it would be best if they had url encoding.

You might as well `url_encode` everything after the body= part, rather then try to do it piecemeal